marketplace/emoji-tablet/behaviors/Edible.js

```diff
@@ -0,0 +1,107 @@
+(function () {
+    var NOM_URL = "https://hifi-content.s3.amazonaws.com/elisalj/emoji_scripts/behaviors/sounds/nom-nom.wav";
+    var NOM = SoundCache.getSound(Script.resolvePath(NOM_URL));
+
+    var _this = this;
+    _this.preload = function (entityID) {
+        _this.entityID = entityID;
+        var props = Entities.getEntityProperties(entityID);
+        print("Loading properties");
+    }
+
+    var checkIfNearHead = function () {
+        var pos = Entities.getEntityProperties(_this.entityID).position;
```

Comment:
Spell out name of variable; only request position property in getEntityProperties call